README.md

```diff
@@ -110,78 +63,22 @@ sudo rm -rf **\~/.solcx**
 sudo rm -rf **\~/.vvm**
 ```
 
-## Environment Variables:
-
-Some plugins require environment variables to connect to their external systems, such project IDs or API keys.
-Follow instructions from individual plugin documentations, such as:
-
-* [ape-alchemy](https://github.com/ApeWorX/ape-alchemy/blob/main/README.md#quick-usage)
-* [ape-infura](https://github.com/ApeWorX/ape-infura#readme)
-
-Generally, set environment variables by doing the following:
-
-```bash
-# Used by the `ape-infura` plugin
-export WEB3_INFURA_PROJECT_ID=<YOUR_INFURA_PROJECT_ID>
-# Used by the `ape-alchemy` plugin
-export WEB3_ALCHEMY_API_KEY=<YOUR_ALCHEMY_KEY>
-```
-
-Place these in environment files, such as your `.bashrc` or `.zshrc`.
+# Quick Start 
 
-## Quick Usage
+After you installed Ape, you can type `ape --version` to make sure it works and it is the latest version. There are two ways to interact with Ape:
+- [CLI Reference](https://docs.apeworx.io/ape/latest/index.html)
+- [Python Reference](https://docs.apeworx.io/ape/latest/index.html)
 
-Use `-h` to list all the commands:
+Ape is both a CLI tool as well as a Python SDK. The CLI tool contains all the Ape commands and the Python SDK contains the classes and types needed to compose scripts, console actions, and tests.
 
-```bash
-ape -h
-```
-
-### Projects
-
-When using Ape, you generally will work with a project.
-To quickly get started using ape, generate a project using the `ape init` command:
-
-```bash
-ape init
-```
-
-For more in-depth information about smart-contract projects using the Ape framework, see the [projects guide](https://docs.apeworx.io/ape/stable/userguides/projects.html).
-
-### Accounts
-
-In Ape, you will need accounts to make transactions.
-You can import or generate accounts using the core `accounts` plugin:
-
-```bash
-ape accounts import acc0   # Will prompt for a private key
-ape accounts generate acc1
-```
-
-List all your accounts with the `list` command.
-
-```bash
-ape accounts list
-```
-
-Learn more about accounts in Ape by following the [accounts guide](https://docs.apeworx.io/ape/stable/userguides/accounts.html).
-
-### Plugins
-
-Add any plugins you may need, such as `vyper`.
-
-```bash
-ape plugins list -a
-ape plugins install vyper
-ape plugins list -a
-```
+### **Ape Modular Plugin Sytem:** 
```

Comment:
```suggestion
### **Ape Modular Plugin Sytem:**

```

Needs newlines after headers.
Also, try to remove trailing spaces if you can for the whole doc